src/main/java/me/leoko/advancedban/bungee/BungeeMethods.java

```diff
@@ -144,12 +147,23 @@ public void sendMessage(Object player, String msg) {
 
     @Override
     public boolean hasPerms(Object player, String perms) {
-        return ((CommandSender) player).hasPermission(perms);
+        try {
+            return ((CommandSender) player).hasPermission(perms);
+        } catch (Exception ex) {
```

Comment:
The redisbungee PubSubListener only checks if the player has the perms for all bungee servers, so if the player isn't in the bungee server that is requesting this it gonna return a null.